frontend/src/app/modules/common/copy-to-clipboard/copy-to-clipboard.directive.ts

```diff
@@ -27,16 +27,16 @@
 //++
 
 
-import {Directive, ElementRef, HostListener, Input, OnInit} from "@angular/core";
+import {Component, ElementRef, OnInit} from "@angular/core";
 import {NotificationsService} from "core-app/modules/common/notifications/notifications.service";
 import {ConfigurationService} from "core-app/modules/common/config/configuration.service";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
-import {Component} from "@angular/core";
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
+
+export const copyToClipboardSelector = 'copy-to-clipboard';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/modules/common/copy-to-clipboard/copy-to-clipboard.directive.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>